### PR TITLE
Return early when arsi is disabled.

### PR DESCRIPTION
**Pull request by barker1977**
#17 hasn't yet addresses all the issues we're seeing. 
While we work on the fix we want to disable arsi globally. Without this change, connections are still being made to the database which means the `ActiveRecord::StatementInvalid: Mysql2::Error: Table doesn't exist` error is still surfacing.

@grosser @zenspider @vanchi-zendesk @cubeguerrero

### CODE REVIEW

**Review by grosser — APPROVED** [review]

File: lib/arsi.rb [hunk 1]
```diff
@@ -31,6 +31,7 @@ def sql_check!(sql, relation)
     end
 
     def arel_check!(arel, relation)
+      return if !@enabled
```
Comment (grosser) [inline lib/arsi.rb:34]: FYI
```suggestion
      return unless @enabled
```

File: test/arsi_test.rb [hunk 2]
```diff
@@ -85,6 +85,18 @@
...
+  describe "when arsi is disabled" do
+    before do
+      Arsi::UnscopedSQL.expects(:sql_check!).never
+      Arsi.disable!
+    end
+
+    it "does not call sql_check if disabled" do
+      assert User.where(:password => 'hello').delete_all
+      Arsi.enable!
+    end
```
Comment (grosser) [inline test/arsi_test.rb:97]: better split of concerns:
```suggestion
    around do |t|
      Arsi.disable!
      t.call
    ensure
      Arsi.enable!
    end

    it "does not call sql_check if disabled" do
      Arsi::UnscopedSQL.expects(:sql_check!).never
      assert User.where(password: 'hello').delete_all
    end
```

Comment (barker1977) [reply test/arsi_test.rb]: 💚

Comment (barker1977) [reply test/arsi_test.rb:97]: Oh, `around` is rspec

Comment (grosser) [reply test/arsi_test.rb]: use minitest-around or maxitest

**Review by cubeguerrero — APPROVED** [review]

File: test/arsi_test.rb (hunk 2 above)
Comment (grosser) [reply test/arsi_test.rb]: https://github.com/zendesk/arsi/pull/20

**Pull request merged**